app/app.go

```diff
@@ -205,7 +205,7 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 	// initialized with tokens from genesis accounts.
 	app.mm.SetOrderInitGenesis(genaccounts.ModuleName, supply.ModuleName, distr.ModuleName,
```

Comment:
shouldn't `supply.ModuleName` be removed after genaccounts - before distr?

Reply:
of course it must be removed! thanks!

Reply:
done btw